haystack/database/base.py

```diff
@@ -82,16 +82,83 @@ def __init__(self, question: str,
         :param document_id: the document_store's ID for the returned answer document.
         :param offset_start_in_doc: the answer start offset in the document.
         :param no_answer: whether the question in unanswerable.
-        :param model_id: model_id used for prediction(in-case of user feedback).
+        :param model_id: model_id used for prediction (in-case of user feedback).
         """
-        self.no_answer = no_answer
-        self.origin = origin
         self.question = question
+        self.answer = answer
         self.is_correct_answer = is_correct_answer
         self.is_correct_document = is_correct_document
+        self.origin = origin
         self.document_id = document_id
-        self.answer = answer
         self.offset_start_in_doc = offset_start_in_doc
+        self.no_answer = no_answer
+        self.model_id = model_id
+
+    @classmethod
+    def from_dict(cls, dict):
+        return cls(**dict)
+
+    def to_dict(self):
+        return self.__dict__
+
+    # define __eq__ and __hash__ functions to deduplicate Label Objects
+    def __eq__(self, other):
+        return (isinstance(other, self.__class__) and
+                getattr(other, 'question', None) == self.question and
+                getattr(other, 'answer', None) == self.answer and
+                getattr(other, 'is_correct_answer', None) == self.is_correct_answer and
+                getattr(other, 'is_correct_document', None) == self.is_correct_document and
+                getattr(other, 'origin', None) == self.origin and
+                getattr(other, 'document_id', None) == self.document_id and
+                getattr(other, 'offset_start_in_doc', None) == self.offset_start_in_doc and
+                getattr(other, 'no_answer', None) == self.no_answer and
+                getattr(other, 'model_id', None) == self.model_id)
+
+    def __hash__(self):
+        return hash(self.question +
+                    self.answer +
+                    str(self.is_correct_answer) +
+                    str(self.is_correct_document) +
+                    str(self.origin) +
+                    str(self.document_id) +
+                    str(self.offset_start_in_doc) +
+                    str(self.no_answer) +
+                    str(self.model_id))
+
+
+class Aggregated_Label:
```

Comment:
How about naming it "MultiLabel"? Feels more intuitive to me ...
In any case, the class name should be camel case.